egapro/bin.py

```diff
@@ -111,6 +111,14 @@ async def search(q, verbose=False):
             print(row)
 
 
+@minicli.cli
+async def export(path: Path):
```

Comment:
Do we want a naming that's more precise? We already have one export `dump_dgt`, so maybe we could have
- dump_dgt > export_dgt
- export > export_search

What do you think?